aws/modules/nlb/README.md

```diff
@@ -31,6 +31,7 @@
 | [aws_security_group_rule.allow_nlb_to_nodes_health](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule) | resource |
 | [aws_security_group_rule.allow_nlb_to_nodes_http](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule) | resource |
 | [aws_security_group_rule.allow_nlb_to_nodes_pgwire](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule) | resource |
+| [aws_vpc_security_group_egress_rule.nlb_egress](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_security_group_egress_rule) | resource |
```

Comment:
Adding comment here since the variables file isn't in this PR ... but while we're here, the description for `internal` is incorrect w.r.t. the default, yes? since the default column has `true` 
https://github.com/MaterializeInc/materialize-terraform-self-managed/blob/80281de7fe40ebeec9eb2aed193321306c5a5ef6/aws/modules/nlb/README.md#input_internal